desktop/pipewire.nix

```
@@ -0,0 +1,10 @@
+# Use pipewire instead of PuleseAudio. See https://nixos.wiki/wiki/PipeWire
+{
+  security.rtkit.enable = true;
+  services.pipewire = {
+    enable = true;
+    alsa.enable = true;
+    alsa.support32Bit = true;
```

Probably does not make sense on other architectures than x86:

```suggestion
    alsa.support32Bit = pkgs.stdenv.isx86_64;
```

---

But also, what applications are still 32-bit? I have not any in a while.

---

Good call. I added this for Steam and could be enabled separately.

---

I see. I guess with games we will have 32-bit at least for another decade.